static-site/src/components/ListResources/Showcase.jsx

```diff
@@ -10,18 +12,35 @@ const cardWidthHeight = 160; // pixels
 
 export const Showcase = ({cards, setSelectedFilterOptions}) => {
   if (!cards.length) return null;
+
+  // Generate options for carousel
+  let responsiveOptions = {};
+
+  for (let i = 0; i < cards.length; i++) {
+    const breakpointMin = i * cardWidthHeight;
+    let breakpointMax = (i + 1) * cardWidthHeight;
+
+    // Don't limit max otherwise carousel won't render at all on larger screens
+    if (i == cards.length - 1) {
+      breakpointMax = 999999
+    }
+
+    responsiveOptions[i] = {
+      breakpoint: { max: breakpointMax, min: breakpointMin },
+      items: i
+    };
+  }
+
   return (
     <div>
       <Byline>
         Showcase resources: click to filter the resources to a pathogen
       </Byline>
-      <SingleRow>
-        <ShowcaseContainer>
-          {cards.map((el) => (
-            <ShowcaseTile data={el} key={el.name} setSelectedFilterOptions={setSelectedFilterOptions}/>
-            ))}
-        </ShowcaseContainer>
-      </SingleRow>
+      <Carousel responsive={responsiveOptions} renderButtonGroupOutside={true}>
```

Comment:
@jameshadfield re: https://github.com/nextstrain/nextstrain.org/pull/873#pullrequestreview-2069756947

> The arrows currently obscure the titles at the moment (or vice versa?)

There's props to customize the arrows which I'll look into. I tried briefly with `renderButtonGroupOutside` but looks like more options are necessary to get it working properly.